values.yaml

```diff
@@ -1,101 +1,137 @@
 ## for NATS service configuration see #scripts/nats-bootstrap/values-nats.yaml.tmpl
 
-location: sandbox
+image: &image_anchor
+  image:
+    pullPolicy: Always
+    repository:
+      tag: latest
+      url: localhost:5001
+
+env: &env_anchor
+  env:
+    facility: sandbox
+    log_level: debug
+    endpoints:
+      nats:
+        url: nats://nats:4222
+        timeout: 60s
+      otel:
+        authenticate: false
+        url: jaeger:4317
+      fleetdb:
+        authenticate: false
+        url: http://fleetdb:8000
+        oidc_audience_url: # to be filled once oauth is available in the sandbox
+        oidc_issuer_url:   # to be filled once oauth is available in the sandbox
+        oidc_client_id: # to be filled by each service bellow
+        oidc_scopes:    # to be filled by each service bellow
```

Comment:
You might want to refer to the stuff you have below to save people making the same mistake I did. 😀